tests/util/ginkgo/lib.go

```diff
@@ -475,7 +475,7 @@ func (ns NsWrapper) RetrieveStatusFromNodetool(podName string) []NodetoolNodeInf
 	nodeTexts := regexp.MustCompile(`(?m)^.*(([0-9a-fA-F]+-){4}([0-9a-fA-F]+)).*$`).FindAllString(output, -1)
 	nodeInfo := []NodetoolNodeInfo{}
 	for _, nodeText := range nodeTexts {
-		comps := regexp.MustCompile(`[[:space:]]+`).Split(nodeText, -1)
+		comps := regexp.MustCompile(`[[:space:]]+`).Split(strings.TrimSpace(nodeText), -1)
```

Comment:
This is because with C* 4 there is some dangling whitespace at the end of the line. This doesn't happen with C* 3.